demo/temp/gbe-all-plugins-bundle.js

```diff
@@ -14907,6 +14907,7 @@ var CacheDriverPlugin = /** @class */ (function () {
      * callbacks.
```

Comment:
Please revert this file. Since we're getting a release process soon, we can put up with this not being up to date for a little bit longer.